README.md

```diff
@@ -242,3 +242,40 @@ Testing looks something like this:
 
 But in production use you'd expect it more like
    `curl <auth to your proxy> https://ssoproxy.example.com/enroll_user -d @~/.ssh/bob.pub`
+
+### GitHub SSH CA Support
+
+Sharkey supports issuing user certificates that are compatible with GitHub SSH CA format by:
+
+mapping a GitHub username to a SAML identity, and
+including appropriate GitHub username in each certificate
+GitHub supports authentication using SSH certificates for Enterprise Cloud accounts. The only requirement is that certificates include GitHub usernames, so that they can be matched to a particular user.
+
+Sharkey already requires SSO proxy for the user certificate feature. Additionally, the GitHub integration requires that the GitHub organization is configured with SSO (i.e. non-GitHub) access.
+
+An example config with GitHub SSH CA Support enabled can be found in `test/git_server_config.yaml`.
+A GitHub App with read/write access to `Organization:members` is required. 
+
+Sharkey uses a cronjob that runs every 5 minutes to query GitHub for a mapping of SAML identities to GitHub usernames.
```

Comment:
This is probably too detailed and quite likely will become outdated (i.e. already the sync interval is configurable and the db table no longer is called `github`).

I'd just say that once enabled via config, Sharkey will periodically sync the user mappings and stores them in the database. When a request is issued to sign user's public key, Sharkey will check in the database if there's a user mapping, and if one exists it will add GitHub username information to the signed certificate.